pallets/block-rewards/src/lib.rs

```diff
@@ -39,13 +39,15 @@ pub mod weights;
 #[cfg(feature = "runtime-benchmarks")]
 mod benchmarking;
 
-use cfg_primitives::{COLLATOR_GROUP_ID, DEFAULT_COLLATOR_STAKE};
 use cfg_traits::ops::{EnsureMul, EnsureSub};
 pub use cfg_traits::{
 	ops::{EnsureAdd, EnsureAddAssign},
 	rewards::{AccountRewards, CurrencyGroupChange, DistributedRewards, GroupRewards},
 };
-use cfg_types::tokens::CurrencyId as CfgCurrencyId;
+use cfg_types::{
+	ids::{COLLATOR_GROUP_ID, DEFAULT_COLLATOR_STAKE},
+	tokens::{CurrencyId as CfgCurrencyId, StakingCurrency},
+};
```

Comment:
Regarding my first comment. This probably is a long discussion that can be done in another issue, but, I think that in the `cfg_types` crate, we are mixing runtime things with common generic pallet things. In this case, our `CurrencyId` is almost sure part of how our runtime works.